tests/unit/src/test/scala/tests/decorations/SyntheticDecorationsLspSuite.scala

```diff
@@ -227,7 +201,7 @@ class SyntheticDecorationsLspSuite extends BaseLspSuite("implicits") {
            |  }
            |  implicit val andy : String = "Andy"
            |  hello()
-           |  ("1" + "2").map[Char, String](c: Char => c.toUpper)
+           |  ("1" + "2").map(c: Char => c.toUpper)
```

Comment:
Is there no type parameter anymore? Could we change it to test case with a parameter?

Reply:
Ah good catch, yes I changed this to `toDouble` which will include a parameter.